src/components/ExploreList.jsx

```diff
@@ -0,0 +1,48 @@
+import React, {useState, useEffect} from 'react'
+import { Item, Segment, Header, Message } from 'semantic-ui-react'
+import {getExploreArticles} from '../modules/dataCenter'
```

Comment:
Why do you call the file `dataCenter`? Please mind your naming standard and name it after the responsibility it has and then add module, service och helper to the name.